resources/home/dnanexus/run_workflows/utils/manage_dict.py

```
@@ -279,6 +280,7 @@ def add_other_inputs(
     project_name,
     sample=None,
     sample_prefix=None,
+    job_outputs_dict={},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)